kvmagent/kvmagent/plugins/host_model_mount_plugin.py

```
@@ -569,14 +626,10 @@ def mount_model_center(self, req):
 
-            # Build mount point path with boundary check
-            mount_point = os.path.abspath(os.path.join(MODEL_MOUNT_BASE, cmd.modelCenterUuid))
-            base_real = os.path.realpath(MODEL_MOUNT_BASE)
-            target_real = os.path.realpath(mount_point)
-
-            # Path boundary check: ensure the resolved mount point stays under MODEL_MOUNT_BASE
-            if not (target_real == base_real or target_real.startswith(base_real + os.sep)):
-                rsp.error = "invalid mount path"
+            try:
+                mount_point = _resolve_model_mount_point(cmd)
+            except Exception as path_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)